liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/server/DevMojo.java

```diff
@@ -574,11 +548,79 @@ private void runTestMojo(String groupId, String artifactId, String phase) throws
             } else {
                 log.debug("Summary file doesn't exist");
             }
+        } else if (phase.equals("failsafe-report-only")) {
+            Plugin failsafePlugin = getPlugin("org.apache.maven.plugins", "maven-failsafe-plugin");
+            Xpp3Dom failsafeConfig = getPluginConfig(failsafePlugin, "integration-test");
+            if (failsafeConfig != null) {
+                Xpp3Dom reportsDirectoryElement = failsafeConfig.getChild("reportsDirectory");
```

Comment:
I think we should also check whether the failsafeConfig actually has a reportsDirectory.